### PR TITLE
COR-649: rename gender to sex

### PR DESCRIPTION
**Pull request by internetti**

### CODE REVIEW

File: internal/api/gender.go [hunk 1]
```diff
@@ -8,83 +8,83 @@ import (
...
+	SexUnspecified Sex = ""
 )
 
-func AllGenders() containers.Set[Gender] {
-	return containers.NewSet[Gender](
-		GenderMale,
-		GenderFemale,
-		GenderOther,
-		GenderPreferNotToSay,
+func AllSexs() containers.Set[Sex] {
```
Comment (ludydoo) [inline internal/api/gender.go:22]:
```suggestion
func AllSexes() containers.Set[Sex] {
```

File: internal/db/sql_individual_list.go [hunk 2]
```diff
@@ -47,7 +47,7 @@ func newGetAllIndividualsSQLQuery(driverName string, options api.ListIndividuals
 		withFreeField4(options.FreeField4).
 		withFreeField5(options.FreeField5).
 		withFullName(options.FullName).
-		withGenders(options.Genders).
+		withSexs(options.Sexs).
```
Comment (ludydoo) [inline internal/db/sql_individual_list.go:50]:
```suggestion
		withSexs(options.Sexes).
```

File: web/static/bootstrap-icons.css [hunk 3]
```diff
@@ -1365,10 +1365,10 @@ url("/static/bootstrap-icons.woff?08efbba7c53d8c5413793eecb19b20bb") format("wof
 .bi-file-earmark-pdf::before { content: "\f63e"; }
 .bi-file-pdf-fill::before { content: "\f63f"; }
 .bi-file-pdf::before { content: "\f640"; }
-.bi-gender-ambiguous::before { content: "\f641"; }
-.bi-gender-female::before { content: "\f642"; }
-.bi-gender-male::before { content: "\f643"; }
-.bi-gender-trans::before { content: "\f644"; }
+.bi-sex-ambiguous::before { content: "\f641"; }
```
Comment (ludydoo) [inline web/static/bootstrap-icons.css:1368]: I wouldn't rename the bootstrap icons constants

File: internal/api/gender.go (hunk 1 above)
Comment (internetti) [reply internal/api/gender.go:22]: That's what you get for not manually replacing everything I guess ^^

**Review by ludydoo — APPROVED** [review]

**Pull request merged**